Compile errors when building with features = ["metal"]

I'm new to Rust and to this project so maybe I'm doing something wrong, but I tried this:

```
$ cargo init
$ cat >> Cargo.toml
[dependencies.skia-safe]
version = "0"
features = ["metal"]
$ cargo build
```

and got several compile errors, e.g.

```
error[E0432]: unresolved import `skia_bindings::GrBackendDrawableInfo`
 --> /Users/nornagon/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-safe-0.27.1/src/gpu/backend_drawable_info.rs:6:5
  |
6 | use skia_bindings::GrBackendDrawableInfo;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `GrBackendDrawableInfo` in the root

error[E0432]: unresolved import `skia_bindings::GrBackendApi`
 --> /Users/nornagon/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-safe-0.27.1/src/gpu/types.rs:1:9
  |
1 | pub use skia_bindings::GrBackendApi as BackendAPI;
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `GrBackendApi` in the root

error[E0432]: unresolved import `skia_bindings::GrRenderable`
  --> /Users/nornagon/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-safe-0.27.1/src/gpu/types.rs:17:9
   |
17 | pub use skia_bindings::GrRenderable as Renderable;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `GrRenderable` in the root

error[E0432]: unresolved import `skia_bindings::GrMtlTextureInfo`
 --> /Users/nornagon/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-safe-0.27.1/src/gpu/mtl/types.rs:4:5
  |
4 | use skia_bindings::GrMtlTextureInfo;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `GrMtlTextureInfo` in the root

[...]

error: aborting due to 114 previous errors
```

Am I holding it wrong?

--- a/skia-bindings/build_support/skia.rs
+++ b/skia-bindings/build_support/skia.rs
@@ -18,6 +18,7 @@ mod lib {
 mod feature_id {
     pub const GL: &str = "gl";
     pub const VULKAN: &str = "vulkan";
+    pub const METAL: &str = "metal";
     pub const TEXTLAYOUT: &str = "textlayout";
 }
 
@@ -96,6 +97,26 @@ pub struct Features {
 impl Features {
     pub fn gpu(&self) -> bool {
         self.gl || self.vulkan || self.metal
+    }
+
+    /// Feature Ids used to look up prebuilt binaries.
+    pub fn ids(&self) -> Vec<&str> {
+        let mut feature_ids = Vec::new();
+
+        if self.gl {
+            feature_ids.push(feature_id::GL);
+        }
+        if self.vulkan {
+            feature_ids.push(feature_id::VULKAN);
+        }
+        if self.metal {
+            feature_ids.push(feature_id::METAL);
+        }
+        if self.text_layout {
+            feature_ids.push(feature_id::TEXTLAYOUT);
+        }
+
+        feature_ids
     }
 }
 
@@ -336,16 +357,9 @@ impl BinariesConfiguration {
 
         let mut built_libraries = Vec::new();
         let mut additional_files = Vec::new();
-        let mut feature_ids = Vec::new();
+        let feature_ids = features.ids();
 
-        if features.gl {
-            feature_ids.push(feature_id::GL);
-        }
-        if features.vulkan {
-            feature_ids.push(feature_id::VULKAN);
-        }
         if features.text_layout {
-            feature_ids.push(feature_id::TEXTLAYOUT);
             additional_files.push(ICUDTL_DAT.into());
             built_libraries.push(lib::SKPARAGRAPH.into());
             built_libraries.push(lib::SKSHAPER.into());

